qemu/tests/cfg/tpm_verify_device.cfg

```diff
@@ -30,6 +30,10 @@
         pattern_output_wmic = IsActivated_InitialValue=TRUE;IsEnabled_InitialValue=TRUE
         cmd_get_tpm = 'powershell -command "get-tpm"'
         pattern_output_get_tpm = TpmPresent\s+:\s+True;TpmReady\s+:\s+True
+        cmd_check_event_log = 'powershell -command "Get-EventLog -LogName System -EntryType Error -Source TPM"'
+        pattern_output_event_log = 'No matches found'
```

Comment:
What if the pattern was changed? This is controlled by third party. Do we have a more accurate way to check if there is any failure, like the error type or return value?

Reply:
We can define different Entrytypes  or other parameters in our command, it depends on the requirements, or what kind of message we are interested in.  If there are some records found by the powershell command, it means there are some error message in eventlog for vtpm device. Return value of the command depends on the  expected output. For now, if no product bug, this command returns 1.